com_028_rw1_effect_of_ag_prices_on_commodity_prices/README.md

```diff
@@ -1,7 +1,16 @@
 ## Effect of Agricultural Policies on Commodity Prices Dataset Pre-processing
 This file describes the data pre-processing that was done to [the Nominal Rate of Protection](http://www.ag-incentives.org/indicator/nominal-rate-protection) for [display on Resource Watch](https://resourcewatch.org/data/explore/641c0a35-f2e5-4198-8ed9-576ea7e9685a).
 
-The data source provided the dataset as one csv file. Years in the 'year' column were converted to datetime objects and stored in a new column 'datetime'. The 'notes' column was removed since it only contains indexes instead of actual data. The column names were converted to lowercase before we uploaded the data to Carto. 
+The data source provided the dataset as one csv file. 
+
+Below, we describe the steps used to reformat the table so that it is formatted correctly to upload to Carto.
+
+1. Years in the 'year' column were converted to datetime objects and stored in a new column 'datetime'. 
+2. A subset was extracted to only retain 'COUNTRY_TOTAL' (aggregate of all products at country level) in the 'category' column.
+3. The 'notes' column was removed since it only contains indexes instead of actual data. 
+4. The 'productcode' column was removed since it contains the same information as the column 'productname'.
+5. The 'source' column was removed since it contains the same information as the column 'sourceversion'
```

Comment:
```suggestion
4.  Remove the 'notes' column since it only contains indexes instead of actual data. 
5. Remove the 'productcode' column since it contains the same information as the column 'productname'.
6. Remove the 'source' column since it contains the same information as the column 'sourceversion'
```